xformers/version.py

```diff
@@ -0,0 +1,2 @@
+# noqa: C801
+__version__ = "0.0.15.dev+2815dc0.d20221213"
```

Comment:
This file is autogenerated and shouldn't be committed - but good point: we should add this to the .gitignore in a future PR :)

Reply:
done